CMakeLists.txt

```diff
@@ -62,6 +62,7 @@ include(ROCMHeaderWrapper)
 
 
 set(MIOPEN_ENABLE_SQLITE On CACHE BOOL "")
+set(MIOPEN_ENABLE_HEUR On CACHE BOOL "Enable AI based heuristics")
```

Comment:
It seems that AI heuristics works with gfx908 only for now. We should not disable WTI-based machinery for other targets.